db/Retail/LocalizationDB.lua

```diff
@@ -63902,7 +63902,7 @@ localize(L.HEADER_DESCRIPTIONS, {
 	[-397] = "Hay una batalla 4x4 en curso. Ayuda a la Horda derrotando a los Worgen Stalkers.",
 	[-549] = "Los siguientes conjuntos de objetos se implementaron como recompensas de mejoras de personaje para los receptores del Pergamino de la Resurrección.\n\nPosteriormente, se añadieron a los vendedores del Monte Hyjal y Vash'jir una vez que se retiró el Servicio del Pergamino de la Resurrección.",
 	[-550] = "Este es un evento basado en Campos de batalla que coincide con el inicio de los Juegos Olímpicos de Verano. La única vez que se celebró fue en 2008 para coincidir con los Juegos Olímpicos de Pekín, y aunque parecía haber intención de repetirlo, nunca regresó.",
-	[-552] = "Estos fueron recompensados ​​por iniciar sesión al menos una vez por facción mientras la película de Warcraft aún estaba en los cines.",
+	[-552] = "Estos fueron recompensados por iniciar sesión al menos una vez por facción mientras la película de Warcraft aún estaba en los cines.",
```

Comment:
Can you revert this change? Probably some Unicode change that does not matter but, regardless.